docs/dev/adding_instruments.rst

```diff
@@ -442,6 +462,40 @@ The dictionary now maps the keys to specific values. The values and keys can be
 
 As you have seen, the :func:`Instrument.control <pymeasure.instruments.Instrument.control>` function can be significantly extended by using validators and maps.
 
+.. _boolean-properties:
+
+Boolean properties
+******************
+
+The idea of using maps can be leveraged to implement properties where the user-facing values are booleans, so you can interact in a pythonic way using :code:`True` and :code:`False`:
+
+.. testcode::
+
+    Extreme5000.output_enabled = Instrument.control(
+        "OUTP?", "OUTP %d",
+        """Control the instrument output is enabled (boolean).""",
+        map_values=True,
+        values={True: 1, False: 0},  # the dict values could also be "on" and "off", etc.
+    )
+
+
+.. doctest::
+
+    >>> extreme = Extreme5000("GPIB::1")
+    >>> extreme.output_enabled = True
+    >>> extreme.read()
+    '1'
+    >>> extreme.output_enabled = False
+    >>> extreme.output_enabled
+    False
```

Comment:
How can you read without prior writing?
I guess you ment to show the raw communication.
```suggestion
    >>> extreme.write("OUTP?")
    >>> extreme.read()  # read the response of the device prior to mapping.
    '1'
    >>> extreme.output_enabled = False
    >>> extreme.output_enabled  # read the mapped value
    False
```

Reply:
Indeed. I think this is a bit of an artifact of how our faked instrument works like, and occurs a couple of times more.

Reply:
This will need some deeper changes if we want it to be more representative -- the first line `extreme.output_enabled = True` basically puts a (mapped) `1` into a primitive buffer in `FakeAdapter`, which the `read` consumes without mapping.

If we can't easily adapt `FakeAdapter` to give more meaningful (doctestable) interaction, maybe the easiest thing would be to define a pyvisa-sim device for the handful of examples that use that misleading raw `read()` approach.

Reply:
I'll bump this to another issue, we have been doing this for a long time that way and nobody noticed, and it will just drag this PR out much longer :D

Reply:
I'll resolve this in favour of #735.

Reply:
Ok, resolve it. Somebody has to know the code to find the error, but those don't need the example for the basic communication.